protobuf/mapping.go

```diff
@@ -17,12 +37,35 @@ func (t *ProtoType) Type() Type {
 	return NewNamed(t.Package, t.Name)
 }
 
+type Decorator func(*Package, *Message, *Field)
+type Decorators []Decorator
+
+func NewDecorators(fns ...func(*Package, *Message, *Field)) Decorators {
+	var decorators Decorators
+
+	for _, fn := range fns {
+		decorators = append(decorators, Decorator(fn))
+	}
+
+	return decorators
+}
+
+func (d Decorator) Run(p *Package, m *Message, f *Field) {
```

Comment:
why adding `Run` to decorator?

you can do `decorator(p, m, f)` instead of `decorator.Run(p, m, f)`